src/Local/Toolstack/Symfony.php

```diff
@@ -48,6 +29,7 @@ public function install()
         $symfonyResources = CLI_ROOT . '/resources/symfony';
 
         // Create and symlink configuration files.
+        // @todo are these actually useful?
```

Comment:
So I'm going to need some help with answering this.

There may be other useful things we can do for Symfony users.